bigquery_etl/public_data/publish_gcs_metadata.py

```diff
@@ -164,6 +167,16 @@ def publish_table_metadata(table_metadata, bucket):
             fout.write(json.dumps(metadata.files_metadata_to_json(), indent=4))
 
 
+def publish_last_modified(table_metadata, bucket):
```

Comment:
We're using a mix of `last_updated` and `last_modified`. My preference would be for `last_modified` as that seems more familiar, but the important thing is to be consistent everywhere we use the term.